country-a-service/cda-generator/src/test/java/dk/sundhedsdatastyrelsen/ncpeh/cda/EPrescriptionL3GeneratorTest.java

```diff
@@ -99,4 +108,14 @@ public void testCdaValidity(String cpr) throws Exception {
 //            );
         }
     }
+
+    record DrugInfo(String packageFormCode, String numberOfSubPackages, String manufacturerOrganization) {
```

Comment:
This seems like a reasonable abstraction.   Should we use it beyond these tests?  I.e. as a field on `EPrescriptionL3Input`?